app/views/api/v1/validators/_validator.json.jbuilder

```diff
@@ -6,7 +6,11 @@ json.extract! validator,
               :details, :created_at, :updated_at
 
 score = validator.score
-ip = Ip.find_by(address: score.ip_address) if score.present?
+ip = if @ips
```

Comment:
Actually, this is hard to understand what should happen in what case. Tune up the syntax or extract the condition to a method.